components/blitz/src/omero/gateway/facility/BrowseFacility.java

```diff
@@ -130,7 +130,7 @@
     public Collection<DataObject> getHierarchy(SecurityContext ctx, Class rootType,
             List<Long> rootIDs, Parameters options)
             throws DSOutOfServiceException, DSAccessException {
-        if (rootType == null || CollectionUtils.isEmpty(rootIDs))
+        if (rootType == null)
```

Comment:
The `rootIDs.isEmpty()` case will go okay?

Reply:
Yes, same behaviour as roodIDs == null (get all).

Reply:
Aha, maybe also mention in Javadoc, just add "or empty" or something.